packages/python-packages/api-stub-generator/setup.py

```diff
@@ -29,7 +29,10 @@
     author_email="azuresdkengsysadmins@microsoft.com",
     license="MIT License",
     packages=find_packages(),
-    install_requires=["astroid"],
+    install_requires=[
+        "astroid",
```

Comment:
Is latest astroid is causing any issue, then we can pin astroid version.

Reply:
I don't think that's the cause. Their last release was Jan 9th and this started failing yesterday.